packages/codegen-ui-react/package.json

```diff
@@ -32,7 +32,9 @@
   },
   "dependencies": {
     "@aws-amplify/codegen-ui": "2.13.1",
+    "@types/pluralize": "^0.0.29",
```

Comment:
this should be a dev dependency, we don't need types at runtime

Reply:
Moved to dev dependency